chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -47,7 +47,7 @@
 
   bash "disable_openstack_services" do
     code <<-EOF
-      for i in $(systemctl list-units openstack* --no-legend | cut -d" " -f1) \
+      for i in $(systemctl list-units openstack* --no-legend | cut -d" " -f1 | grep -Ev 'keystone|neutron') \
```

Comment:
Yes exactly, it doesn't make sense to keep them in line for this reason.  As with the others, they should be disabled anyway, but it's good to make sure.